public/css/docs.css

```diff
@@ -1269,3 +1247,542 @@ footer .container {
 
 
 /*-------------------------/.MARGIN/PADDING-------------------------*/
+
+html {
+    font-size: 100%;
+    -ms-text-size-adjust: 100%;
+    -webkit-text-size-adjust: 100%
+}
+
+body {
+    margin: 0;
+    padding: 0
+}
+
+pre,
+code,
+address,
+caption,
+th,
+figcaption {
+    font-size: 1em;
+    font-weight: normal;
+    font-style: normal
+}
+
+fieldset,
+iframe,
+img {
+    border: none
+}
+
+caption,
+th {
+    text-align: left
+}
+
+table {
+    border-collapse: collapse;
+    border-spacing: 0
+}
+
+article,
+aside,
+footer,
+header,
+nav,
+main,
+section,
+summary,
+details,
+hgroup,
+figure,
+figcaption {
+    display: block
+}
+
+audio,
+canvas,
+video,
+progress {
+    display: inline-block;
+    vertical-align: baseline
+}
+
+button {
+    font: inherit;
+    vertical-align: middle
+}
+
+*,
+*:before,
+*:after {
+    -moz-box-sizing: border-box;
+    -webkit-box-sizing: border-box;
+    box-sizing: border-box;
+}
+
+html,
+body {
+    height: 100%
+}
+
+body {
+    font-family: "Helvetica Neue", Helvetica, Arial, Verdana, sans-serif;
+    font-size: 18px;
+    line-height: 1.5;
+    color: #333;
+    font-weight: 300
+}
+
+body pre,
+body code {
+    font-family: Menlo, Consolas, Monaco, "Courier New", monospace, serif
+}
+
+p {
+    margin-bottom: 1.5em;
+}
+
+h1 {
+    margin-bottom: 0.5em;
+    font-size: 3em;
+    font-weight: 200;
+    line-height: 1;
+    padding-top: 1em;
+}
+
+h1.active+.main-nav {
+    border-top: 1px solid #333
+}
+
+h2 {
+    margin-bottom: 0.5em;
+    font-size: 2.5em;
+    font-weight: 200;
+    line-height: 1;
```

Comment:
I have noticed this sometimes causes the link and loop of the lowercase g to overflow the border
<img width="421" alt="screen shot 2017-12-05 at 6 02 54 pm" src="https://user-images.githubusercontent.com/159591/33635664-d8ed6f48-d9e6-11e7-98f9-e43525fcf821.png">

something like line-height: 1.2 should be enough to handle this